Breeze.UI/src/app/setup/create/confirm-mnemonic/secret-word-index-generator.ts

```diff
@@ -0,0 +1,18 @@
+export class SecretWordIndexGenerator {
+    
+    private readonly textPrefix = 'Word °';
+    readonly index1 = SecretWordIndexGenerator.getRandom();
+    readonly index2 = SecretWordIndexGenerator.getRandom(this.index1);
+    readonly index3 = SecretWordIndexGenerator.getRandom(this.index1, this.index2);
+    readonly text1 = `${this.textPrefix}${this.index1 +1 }`;
```

Comment:
Do these have to be in order? Like 2, 7, 9 and not 5, 1, 4?

Reply:
I understand following chat